### ISSUE TITLE
Uncaught Error: Cannot find module 'commonmark'

### ISSUE THREAD

**Issue by ghost** [FoldingText/foldingtext-markdown#2]
[Enter steps to reproduce below:]
1. Select any sections of a FTML document, or all, and choose FoldingText Markdown: Copy Markdown from the Command Palette
2. The same type of error occurs when copying from a FoldingText document then attempting to Paste Markdown from the Command Palette

**Atom Version**: 0.207.0
**System**: Mac OS X 10.10.3
**Thrown From**: [foldingtext-markdown](https://github.com/FoldingText/foldingtext-markdown) package, v1.3.0
### Stack Trace

Uncaught Error: Cannot find module 'commonmark'

```
At module.js:330

Error: Cannot find module 'commonmark'
  at Module._resolveFilename (module.js:328:15)
  at Function.Module._resolveFilename (/Applications/Atom.app/Contents/Resources/app.asar/src/module-cache.js:383:52)
  at Function.Module._load (module.js:270:25)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.<anonymous> (/Users/vsp/.atom/packages/foldingtext-markdown/lib/foldingtext-to-markdown.coffee:1:14)
  at Object.<anonymous> (/Users/vsp/.atom/packages/foldingtext-markdown/lib/foldingtext-to-markdown.coffee:1:1)
  at Module._compile (module.js:452:26)
  at Object.requireCoffeeScript (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/coffee-cash/lib/coffee-cash.js:85:19)
  at Module.load (module.js:347:32)
  at Function.Module._load (module.js:302:12)
  at Module.require (module.js:357:17)
  at require (module.js:376:17)
  at Object.module.exports.FoldingMarkdown.copyMarkdown (/Users/vsp/.atom/packages/foldingtext-markdown/lib/foldingtext-markdown.coffee:49:32)
  at ft-outline-editor.foldingTextSubscriptions.add.atom.commands.add.foldingtext-markdown:copy-markdown (/Users/vsp/.atom/packages/foldingtext-markdown/lib/foldingtext-markdown.coffee:25:49)
  at CommandRegistry.module.exports.CommandRegistry.handleCommandEvent (/Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:238:29)
  at /Applications/Atom.app/Contents/Resources/app.asar/src/command-registry.js:3:61
  at CommandPaletteView.module.exports.CommandPaletteView.confirmed (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/command-palette/lib/command-palette-view.js:159:32)
  at CommandPaletteView.module.exports.SelectListView.confirmSelection (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:338:21)
  at space-pen-li.<anonymous> (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/atom-space-pen-views/lib/select-list-view.js:139:19)
  at HTMLOListElement.jQuery.event.dispatch (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4435:9)
  at HTMLOListElement.elemData.handle (/Applications/Atom.app/Contents/Resources/app.asar/node_modules/jquery/dist/jquery.js:4121:28)

```
### Commands

```
     -1:22.9.0 core:select-down (atom-text-editor.editor.is-focused)
     -1:16.3.0 core:copy (atom-text-editor.editor.is-focused)
     -0:11.2.0 core:select-all (span.ft-body-text)
     -0:08.6.0 command-palette:toggle (input.)
     -0:01.2.0 foldingtext-markdown:copy-markdown (input.)
```
### Config

``` json
{
  "core": {
    "themes": [
      "one-light-ui",
      "one-light-syntax"
    ],
    "disabledPackages": [
      "typewriter-sounds"
    ]
  }
}
```
### Installed Packages

``` coffee
# User
foldingtext-for-atom, v0.4.4
foldingtext-markdown, v1.3.0

# Dev
No dev packages
```

**Comment by jessegrosjean** [comment]
I couldn't seem to reproduce this, but I think it should be fixed by 44391eb

**Comment by ghost** [comment]
It works fine after loading 1.3.1. Thanks.
